src/can_protocol.py

```diff
@@ -198,6 +198,31 @@ def update_protocols_on_startup(protocols):
         session.close()
 
 
+def publish_protocol_message_to_can_module(data):
```

Comment:
Where is the value processing done (multiplire, divisor, offset...)? On Cloud already?
If not, it should be covered on the gateway.

Reply:
Resolved issue in latest commit.